xray/src/generation.rs

```diff
@@ -96,6 +49,15 @@ arg_enum! {
     }
 }
 
+arg_enum! {
+    #[derive(Debug)]
+    #[allow(non_camel_case_types)]
+    pub enum ColormapArgument {
+        jet,
+        purpley,
```

Comment:
maybe `purplish`?

Reply:
how about `purplig`? ;-)

Reply:
Haha, honestly, purplish exists, the others don't, so I would opt for the existing one...